client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/LocalClusterClient.kt

```diff
@@ -0,0 +1,26 @@
+package uk.ac.ebi.biostd.client.cluster.api
+
+import arrow.core.Try
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import uk.ac.ebi.biostd.client.cluster.model.Job
+import uk.ac.ebi.biostd.client.cluster.model.JobSpec
+import java.util.UUID
+
+class LocalClusterClient : ClusterOperations {
+    override suspend fun triggerJob(jobSpec: JobSpec): Try<Job> {
+        return withContext(Dispatchers.IO) {
+            Runtime.getRuntime().exec(jobSpec.command)
```

Comment:
I found the process builder nicer.  (I used this one in the RemoteExtSubmissionSubmitter)